docs/Installing.asciidoc

```diff
@@ -624,6 +626,59 @@ This will allow the workers to download the assets from the webUI and use them
 locally. If +TESTPOOLSERVER+ is set tests and needles will also be cached by the
 worker.
 
+[[development-setup]]
+== Development setup
```

Comment:
Can you please move this part to "Contributing.asciidoc" to not confuse pure users?

Reply:
Ok, I'll have to rewrite parts of the documentation for this (e.g. some references don't make any sense otherwise). I don't see how that would confuse people. The first sentence in the section clearly states the purpose of the section.

Reply:
But I guess it is worth it. By moving things around a little we can actually save some cross references and it gets more compact.